_data/projects/Spruce.yml

```diff
@@ -0,0 +1,10 @@
+name: Spruce
+desc: An open source social media platform built with Node.js and MongoDB
+site: https://github.com/DivySrivastava/spruce
+tags:
+- javascript
+- nodejs
+- mongodb
+upforgrabs:
+  name: beginner
+  link: https://github.com/DivySrivastava/spruce
```

Comment:
Give the link to your label that you want to use, for example - https://github.com/DivySrivastava/spruce/labels/good%20first%20issue